qdrant-landing/content/documentation/examples/databricks.md

```diff
@@ -0,0 +1,173 @@
+        sparse_vector = next(sparse_model.embed(row.sentence2))
+
+        yield [
+            row.sentence1,  # 1st column: original text
+            row.sentence2,  # 2nd column: original text
+            dense_vector.tolist(),  # 3rd column: dense vector
+            sparse_vector.indices.tolist(),  # 4th column: sparse vector indices
+            sparse_vector.values.tolist(),  # 5th column: sparse vector values
+        ]
+```
+
+We're using the [BAAI/bge-small-en-v1.5](https://huggingface.co/BAAI/bge-small-en-v1.5) model for dense embeddings and [prithivida/Splade_PP_en_v1](https://huggingface.co/prithivida/Splade_PP_en_v1) for sparse embeddings.
+
+#### Applying the UDF on our dataframe
+
+Next, let's apply our `vectorize` UDF on our Spark dataframe to generate embeddings.
+
+```python
+embeddings = dataset_df.rdd.mapPartitions(vectorize)
+```
+
+The `mapPartitions()` method returns a [Resilient Distributed Dataset (RDD)](https://www.databricks.com/glossary/what-is-rdd) which should then be converted back to a Spark dataframe.
+
+#### Building the new Spark dataframe with the vectorized data
+
+We'll now create a new Spark dataframe (`embeddings_df`) with the vectorized data using the specified schema.
+
+```python
+from pyspark.sql.types import StructType, StructField, StringType, ArrayType, FloatType, IntegerType
+
+# Define the schema for the new dataframe
+schema = StructType([
+    StructField("sentence1", StringType()),
+    StructField("sentence2", StringType()),
+    StructField("dense_vector", ArrayType(FloatType())),
+    StructField("sparse_vector_indices", ArrayType(IntegerType())),
+    StructField("sparse_vector_values", ArrayType(FloatType()))
+])
+
+# Create the new dataframe with the vectorized data
+embeddings_df = spark.createDataFrame(data=embeddings, schema=schema)
+```
+
+### Uploading the data to Qdrant
+
+- **Create a Qdrant collection:**
+  - [Follow the documentation](/documentation/concepts/collections/#create-a-collection) to create a collection with the appropriate configurations. Here's an example request to support both dense and sparse vectors:
+
+  ```json
+  PUT /collections/{collection_name}
+  {
+    "vectors": {
+      "dense": {
+        "size": 384,
+        "distance": "Cosine"
+      }
+    },
+    "sparse_vectors": {
+      "sparse": {}
+    }
+  }
+  ```
+
+- **Upload the dataframe to Qdrant:**
+
+```python
+embeddings_df.write.format("io.qdrant.spark.Qdrant").option(
+    "qdrant_url",
+    "<QDRANT_GRPC_URL>",
+).option("api_key", "<QDRANT_API_KEY>").option(
+    "collection_name", "<QDRANT_COLLECTION_NAME>"
+).option(
+    "vector_fields", "dense_vector"
+).option(
+    "vector_names", "dense"
+).option(
+    "sparse_vector_value_fields", "sparse_vector_values"
+).option(
+    "sparse_vector_index_fields", "sparse_vector_indices"
+).option(
+    "sparse_vector_names", "sparse"
+).option(
+    "schema", embeddings_df.schema.json()
+).mode(
+    "append"
+).save()
+```
```

Comment:
Isn't there any way to set all the options at once?

Reply:
No. Only passing one option at a time is supported.
https://spark.apache.org/docs/2.3.0/sql-programming-guide.html#manually-specifying-options

In case of the Qdrant connector, these are the options.
https://qdrant.tech/documentation/frameworks/spark/#configuration-options

Reply:
Okay, thanks for clarifying it!